style/src/test/java/org/jline/style/StyleExpressionTest.groovy

```diff
@@ -26,7 +26,7 @@ class StyleExpressionTest
     @Before
     void setUp() {
         super.setUp()
-        this.underTest = new StyleExpression(new StyleResolver(source, 'test'))
+        this.underTest = new StyleExpression()
```

Comment:
Some tests do leverage the style source, but this specific test does not, so I thought it made sense to not use it at all.  You're thinking about if we add more tests to this class that would leverage the style source ?
I can revert if you want.